### PR TITLE
[Hub Generated] Review request for Microsoft.KeyVault to add version stable/7.3

### PR DESCRIPTION
**Pull request by susolank**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/susolank/azure-rest-api-specs/dev-keyvault-Microsoft.KeyVault-7.3...main/).
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document:
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json [hunk 1]
```diff
@@ -1473,7 +1473,7 @@
         },
         "exportable": {
           "type": "boolean",
-          "description": "Indicates if the private key can be exported."
+          "description": "Indicates if the private key can be exported. Release policy must be provided when creating 1st version of exportable key."
```
Comment (jackrichins) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json:1476]: Once you mark a key exportable = true, can you later set exportable = false?

Comment (susolank) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json:1476]: No, value for exportable flag can never be changed. Once, the 1st version of a key has exportable set to true, then all new versions of the key have to have same value for exportable.

Exportable is a global property that applies to all versions of a key.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json [hunk 2]
```diff
@@ -1589,7 +1589,7 @@
       "properties": {
         "exportable": {
           "type": "boolean",
-          "description": "Indicates if the private key can be exported."
+          "description": "Indicates if the private key can be exported. Release policy must be provided when creating 1st version of exportable key."
```
Comment (dalzhao) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json:1592]: Release policy must be provided when creating THE 1st version of AN exportable key.

Comment (susolank) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json]: Fixed

**Review by mikekistler — APPROVED** [review]
Looks good! 👍

**Review by dalzhao — APPROVED** [review]

**Pull request merged**